public/docs/dart/latest/guide/architecture.jade

```diff
@@ -1,451 +1,51 @@
-  which has the `<hero-detail>` tag.
-  That tag adds a custom element representing a component we haven't seen yet,
-  a `HeroDetailComponent`.
-
-  The `HeroDetailComponent` is a *different* component than the `HeroListComponent` we've seen.
-  The `HeroDetailComponent` (code not shown) presents facts about a particular hero, the
-  hero that the user selects from the list presented by the `HeroListComponent`.
-  The `HeroDetailComponent` is a **child** of the `HeroListComponent`.
-
-figure
-  img(src="/resources/images/devguide/architecture/component-tree.png" alt="Metadata" align="left" style="width:300px; margin-left:-40px;margin-right:10px" )
-:marked
-  Notice how `<hero-detail>` rests comfortably among native HTML elements.
-  We can and _will_ mix our custom components with native HTML in the same layouts.
-
-  In this manner we'll compose complex component trees to build out our richly featured application.
-<br clear="all">
-
-.l-main-section
-<a id="metadata"></a>
-:marked
-  ## Metadata
-figure
-  img(src="/resources/images/devguide/architecture/metadata.png" alt="Metadata" align="left" style="width:150px; margin-left:-40px;margin-right:10px" )
-:marked
-<p style="padding-top:10px">Metadata tells Angular how to process a class.</p>
-<br clear="all">
-:marked
-  [Looking back at the code](#component-code) for `HeroListComponent`, we see that it's just a class.
-  There is no evidence of a framework, no "Angular" in it at all.
-
-  In fact, it really is *just a class*. It's not a component until we *tell Angular about it*.
-
-  We tell Angular that `HeroListComponent` is a component by attaching **metadata** to the class.
-
-  In Dart, we attach metadata by using an **annotation**.
-  Here's some metadata for `HeroListComponent`:
-
-+makeExample('architecture/dart/lib/hero_list_component.dart', 'metadata', 'lib/hero_list_component.dart')
-:marked
-  Here we see the `@Component` annotation, which (no surprise) identifies the class
-  immediately below it as a component class.
-
-  Annotations often have configuration parameters.
-  The `@Component` annotation takes parameters to provide the
-  information Angular needs to create and present the component and its view.
-
-  Here we see a few of the possible `@Component` parameters:
+    Angular 2 for Dart is published as the `angular2` package, which
+    (like many other Dart packages) is available via the Pub tool.
 
-  * `selector`: A CSS selector that tells Angular to create and insert an instance of this component
-  where it finds a `<hero-list>` tag in *parent* HTML.
-  For example, if an app's  HTML contains `<hero-list></hero-list>`, then
-  Angular inserts an instance of the `HeroListComponent` view between those tags.
+block dart-terminology
+  .callout.is-helpful
+    header Dart difference: Module is a compilation unit
+    :marked
+      The term "module" used in this guide refers to a
+      Dart _compilation unit_, i.e., a _library_ or _part_.
```

Comment:
Fixed as you suggested with tweaks we agree to elsewhere (in particular clarifying that a Dart package is also a module).